Felienne/hedy
# Add error percentage to admin /stats page #1668

Change per level charts of admin /stats page to hold absolute program runs and error rates per level #1668


**Description**

- Consolidate the successful and failed program runs per level into a single chart as stacked bars.
- Add error rate per level in a separate line chart.

**Fix for**

#1668 

**How to test**

- Run hedy locally, make sure you are logged as an admin, and generate some program runs.
- Open the /stats page and see inspect the change in the first couple of charts. Do they present the data better?
